src/EventHandler.ts

```diff
@@ -20,11 +20,13 @@ export class Event {
         this.callbacks.push(callback);
     }
 
-    executeCallback(params?: any | any[]) {
+    executeCallback(params: any[]) {
```

Comment:
@fanwgwg May I know why is this changed? The function should be able to take in array parameters as it is.